src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars_star"></div>
```

Comment:
following BEM methodology we should use `__` for block elements
```suggestion
      <div class="stars__star"></div>
```